app/models/user.py

```diff
@@ -0,0 +1,6 @@
+from pydantic import BaseModel
+from typing import List, Optional
+
+class User(BaseModel):
```

Comment:
Where will this be saved in the database?

Reply:
Users are saved to the UserTable, a second table in the database